examples/azure-functions-hello-world/az_deploy.out

```diff
@@ -0,0 +1,12 @@
+$ func azure functionapp publish bal-bbe --script-root target/azure_functions
+Getting site publishing info...
+Creating archive for current directory...
+Uploading 28.67 MB [##############################################################################]
+Upload completed successfully.
+Deployment completed successfully.
+Syncing triggers...
+Functions in bal-bbe:
+    get-hello - [httpTrigger]
+        Invoke url: https://bal-bbe.azurewebsites.net/hello
+
+    timer - [timerTrigger]
```

Comment:
The timer function shouldn't be here. Recheck all outputs of commands please

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-distribution/pull/4768.